packages/cli/generators/app/templates/src/migrate.ts.ejs

```diff
@@ -6,8 +6,9 @@ export async function migrate(args: string[]) {
 
   const app = new <%= project.applicationName %>();
   await app.boot();
+<% if (project.repositories) { -%>
```

Comment:
I'm wondering if it would be better if we don't have this file generated if the `repository` option is not selected during project scaffolding. 
@strongloop/loopback-maintainers, thoughts?

Reply:
@dhmlau that would also be my suggestion since this file does not do anything if `app.migrateSchema` is not included

Reply:
+1

Can you @pktippa please rework your pull request per the suggestion above?